org.eclipse.xtext.ide.tests/META-INF/MANIFEST.MF

```diff
@@ -17,6 +17,9 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.lsp4j;bundle-version="[0.19.0,0.20.0)";resolution:=optional,
  org.eclipse.lsp4j.jsonrpc;bundle-version="[0.19.0,0.20.0)";resolution:=optional,
  org.eclipse.xtext.testlanguages,
- org.eclipse.xtext.testlanguages.ide
+ org.eclipse.xtext.testlanguages.ide,
+ org.eclipse.emf.mwe2.lib;bundle-version="2.14.0",
+ org.aopalliance;bundle-version="1.0.0",
+ com.google.gson;bundle-version="2.9.1"
```

Comment:
isnt this a transitive dep?

Reply:
Unless they are re-exported in the MANIFEST by other projects I guess they're not (the project did not compile)

Reply:
might be that gradle behaves differently.
do you know what the error message was?
something along required by class file?

Reply:
The tests were failing at runtime with NoClassFoundError